backend/package.json

```diff
@@ -15,29 +15,13 @@
     "dev": "nodemon --exec babel-node src/ -e js,gql",
     "dev:debug": "nodemon --exec babel-node --inspect=0.0.0.0:9229 src/ -e js,gql",
     "lint": "eslint src --config .eslintrc.js",
-    "test": "cross-env NODE_ENV=test NODE_OPTIONS=--max-old-space-size=8192 jest --forceExit --detectOpenHandles --runInBand --coverage --logHeapUsage",
+    "test": "cross-env NODE_ENV=test jest --runInBand --coverage --forceExit --detectOpenHandles",
```

Comment:
We needed that max old space size option here, because sometimes we ha memory leeks ... May be better keep it